test_decorators.py

```diff
@@ -0,0 +1,242 @@
+"""
+Unit test for pep257 module decorator handling.
+
+Use tox or py.test to run the test suite.
+"""
+
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
+
+import pep257
+
+
+class TestParser:
+    """
+    Check parsing of Python source code.
+    """
+
+    def test_parse_class_single_decorator(self):
+        """
+        Class decorator is recorded in class instance.
+        """
+        code = """
+@first_decorator
+class Foo:
+    pass
+        """
```

Comment:
If you're using multi-line strings, indent them like the starting line and then use `textwrap.dedent` (it looks much better this way). You can look at `test_pep257.py` for examples.